testdata/globalconfig.yml

```diff
@@ -60,7 +76,14 @@ metrics:
     - name: response_code
       value_type: 2 # INT64
 quotas:
-- name: RequestCount
-  max_amount: 5
-  expiration:
-    seconds: 1
+  - name: RequestCount
+    max_amount: 5
+    expiration:
+      seconds: 1
+logs:
+  - name: common
```

Comment:
Let's make the name `accesslog.common` and `accesslog.combined`
That we __can__ retire the accessLog `kind`.  Having descriptive names serves similar purpose.